lib/activerecord-tenant-level-security/schema_statements.rb

```diff
@@ -1,22 +1,22 @@
 module TenantLevelSecurity
   module SchemaStatements
-    def create_policy(table_name)
+    def create_policy(table_name, column_name: 'tenant_id')
```

Comment:
It is better to use a something like `partition_key` rather than a general name like `column_name`.

Reply:
Thanks for the more specific variable names, I fixed it in this commit https://github.com/kufu/activerecord-tenant-level-security/pull/22/commits/094779b9c76df5f5a4ed74a59eb30385bb8c5509